spec/lib/lighthouse/benefits_documents/configuration_spec.rb

```diff
@@ -47,22 +47,5 @@
         end
       end
     end
-
```

Comment:
This is not possible situation since it will be blocked before it ever gets to the configuration.